### DISCUSSION TITLE
”Something went wrong” when following a link to a moved issue

### DISCUSSION THREAD

**Discussion by KalleOlaviNiemitalo** [community/community#9900] [Mobile] [answered]
<https://github.com/PowerShell/PowerShell/issues/15843#issuecomment-889740345> links to <https://github.com/dotnet/core/issues/6517>, which has been moved to <https://github.com/dotnet/runtime/issues/56835>. When I try to follow the link using the GitHub 1.44.0 app on iOS 15.2, it just displays ”Something went wrong” instead of following the redirection.

**Comment by brianlovin** [comment, marked as answer]
Thanks for the heads up! We are aware of some issues with redirecting issues via deep links, and I've added this discussion to our tracking issue for our engineers to see 😊
